ubuntu/ubuntu-18.x/ubuntu-18.04-LTS-hpc/install_nvidiagpudriver.sh

```diff
@@ -1,4 +1,40 @@
 #!/bin/bash
 set -ex
 
-$UBUNTU_COMMON_DIR/install_nvidiagpudriver.sh 1804 "b8db71bf86c4b282a96efcb56d6b1bc48df13117b1a46c3e457628fd401587cc"
+# Parameters
+RELEASE_VERSION=1804
+CHECKSUM="749183821ffc051e123f12ebdeb171b263d55b86f0dd7c8f23611db1802d6c37"
```

Comment:
Where does this CHECKSUM get used?

Reply:
Ah yeah it looks like we don't need that. It also appears we don't use the CHECKSUM variable in our common install_nvidiagpudriver.sh script in the master branch anymore either. 

https://github.com/Azure/azhpc-images/blob/master/ubuntu/common/install_nvidiagpudriver.sh#L6